### ISSUE TITLE
docsite: components using tabster modalizer for focus trap are inherently broken

### ISSUE THREAD

**Issue by ling1726** [microsoft/fluentui#26580]
Tabster modalizer lets focus go into the browser window similar to how the native <dialog> element does https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dialog. This means that when rendered inside an iframe it creates the perception that the focus trap in Fluent _**does not work**_ because focus just escapes the example's iframe.

Affected:
* Popover https://react.fluentui.dev/?path=/docs/components-popover--default
* Dialog https://react.fluentui.dev/?path=/docs/components-dialog--default

**Comment by layershifter** [comment]
Closing, as we are using different behavior now.
